docs_app/content/license.md

```diff
@@ -1,23 +0,0 @@
-@title
-@description
-The MIT License
-
-Copyright (c) 2014-2018 Google, Inc.
```

Comment:
I'm not sure we can remove this. We're still using parts of the documentation app that Google built, and therefor the license applies. However, we can probably change this to include "And Contributors" since other contributors have added code to this, and we're also MIT licensed. Or even `Google, Inc., RxJS Team Members and Contributors` or whatever you like.